mitmproxy/addons/cut.py

```diff
@@ -28,6 +28,14 @@ def is_addr(v):
 
 
 def extract(cut: str, f: flow.Flow) -> str | bytes:
+    if (
+        hasattr(f, "websocket")
+        and f.websocket is not None
+        and (cut == "response.content" or cut == "request.content")
+    ):
+        formatted_messages = f.websocket.get_formatted_messages()
+        return formatted_messages
```

Comment:
```suggestion
        return f.websocket.get_formatted_messages()
```

and this needs a test please. :)